install.ps1

```diff
@@ -1,6 +1,53 @@
-$url=(Invoke-WebRequest -Uri https://api.github.com/repos/meshcloud/collie-cli/releases/latest | Select-String -Pattern '.*\"browser_download_url\":\"(.*windows-msvc.exe).*' | ForEach-Object {($_.matches.groups[1]).Value})
+Clear-Host
+Write-Host "`nStarting Installation of Collie ..." -ForegroundColor Green
 
-New-Item -ItemType directory -Path "${HOME}\collie-cli"
-Invoke-WebRequest -Uri $url -OutFile "${HOME}\collie-cli\collie"
+$ProgressPreference = "SilentlyContinue" # Don't remove this! Otherwise it will slow down the Download-Process under PS 5
 
-Write-Host "Please add ${HOME}\collie-cli\ to your path environemnt variable."
\ No newline at end of file
+# Check the latest release to get the exact download URL
+try {
+    $query = Invoke-RestMethod -Uri "https://api.github.com/repos/meshcloud/collie-cli/releases/latest"
+    $downloadUrl = $query.assets | Where-Object { $_.browser_download_url -match "windows-msvc.exe" }
+}
+catch {
+    throw "[ERROR] while downloading Installation File. Check your Internet Connection: $($($_.Exception).Message)!"
```

Comment:
d: are you sure about the capital I and F? Also this variable resolution looks strange to me, this this really correct?
```suggestion
    throw "[ERROR] while downloading installation file. Check your Internet Connection: $(_.Exception.Message)!"
```